src/main/java/org/isf/patient/manager/FileSystemPatientPhotoManager.java

```diff
@@ -0,0 +1,109 @@
+package org.isf.patient.manager;
```

Comment:
The project standard is to include license headers in all new files.

Reply:
done